Assets/HoloToolkit/Utilities/Scripts/DebugPanelControllerInfo.cs

```diff
@@ -28,6 +28,35 @@ private class ControllerState
             public Vector2 TouchpadPosition;
         }
 
+        // Text display label game objects
+        [SerializeField]
+        public GameObject leftInfoTextPointerPosition;
```

Comment:
Can these be TextMesh instead of GameObject? It'd save having to do all the GetComponent<TextMesh>() calls below.